frontend/Dockerfile.e2e

```
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/playwright:v1.40.1-jammy
+FROM mcr.microsoft.com/playwright:v1.44.0-jammy
```

probably fine to drop patch here too
```suggestion
FROM mcr.microsoft.com/playwright:v1.44-jammy
```